target/linux/ramips/image/mt7620.mk

```diff
@@ -1273,6 +1273,16 @@ define Device/xiaomi_miwifi-mini
 endef
 TARGET_DEVICES += xiaomi_miwifi-mini
 
+define Device/bolt_bl201
+  SOC := mt7620a
+  IMAGE_SIZE := 15872k
+  DEVICE_VENDOR := Bolt
+  DEVICE_MODEL := BL 201
+  DEVICE_PACKAGES := kmod-mt76x2 kmod-usb2 kmod-usb-ohci
+  SUPPORTED_DEVICES += bl201
```

Comment:
Why did you add the `bl201` here? Do you need this for the initial flashing?

Reply:
i though it was needed because, i write this based on `../dts/mt7620a_xiaomi_miwifi-mini.dts`. so to fix that just remove `SUPPORTED_DEVICES`?

Reply:
`SUPPORTED_DEVICES` provides alternative names which sysupgrade should also accept.  This is needed when a board was renamed in OpenWrt or mainline OpenWrt is installed using sysupgrade from the vendor firmware if the vendor firmware already uses OpenWrt, but has a different board name.